Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs

```diff
@@ -33,25 +34,23 @@ public ActionMapsView(VisualElement root, StateContainer stateContainer)
 
             CreateSelector(s => new ViewStateCollection<string>(Selectors.GetActionMapNames(s)),
                 (actionMapNames, state) => new ViewState(Selectors.GetSelectedActionMap(state), actionMapNames));
+
+            addActionMapButton.clicked += AddActionMap;
         }
 
         private Button addActionMapButton => m_Root?.Q<Button>("add-new-action-map-button");
 
         public override void RedrawUI(ViewState viewState)
         {
-            m_ListView.bindItem = (element, i) =>
```

Comment:
I assume this is in Rebuild so no longer needed here. Good to remove duplicate code